cmd/fix/blocks/blocks.go

```diff
@@ -0,0 +1,118 @@
+package blocks
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	"github.com/forbole/bdjuno/v2/database"
+	"github.com/forbole/bdjuno/v2/modules"
+	"github.com/forbole/bdjuno/v2/modules/bank"
+	"github.com/forbole/bdjuno/v2/modules/consensus"
+	"github.com/forbole/bdjuno/v2/modules/distribution"
+	"github.com/forbole/bdjuno/v2/modules/gov"
+	"github.com/forbole/bdjuno/v2/modules/history"
+	"github.com/forbole/bdjuno/v2/modules/staking"
+	"github.com/forbole/bdjuno/v2/utils"
+	"github.com/forbole/juno/v2/cmd/parse"
+	junomessages "github.com/forbole/juno/v2/modules/messages"
+	"github.com/forbole/juno/v2/types/config"
+	"github.com/spf13/cobra"
+	tmctypes "github.com/tendermint/tendermint/rpc/core/types"
+)
+
+// blocksCmd returns a Cobra command that allows to fix missing blocks in database
+func blocksCmd(parseConfig *parse.Config) *cobra.Command {
```

Comment:
I think there is an easier way to implement this by using the existing `Worker` structure: 
1. Get the parsing context as you do
2. Build a new `Worker` instance
3. For each height, call the `worker.Process(height)` method returning any error

This will make sure that we call everything needed even if we add new modules in the future and would allow us to remove all the code duplication added to store things inside the database. 

**Note**: to implement this the currently private `process` method must be made public inside Juno.

Reply:
Great! I updated the code and exported `process` method in juno repository